src/pharmpy/tools/iivsearch/tool.py

```diff
@@ -27,7 +33,15 @@
 IIV_STRATEGIES = frozenset(
     ('no_add', 'add_diagonal', 'fullblock', 'pd_add_diagonal', 'pd_fullblock')
 )
-IIV_ALGORITHMS = frozenset(('brute_force', 'brute_force_no_of_etas', 'brute_force_block_structure'))
+IIV_ALGORITHMS = frozenset(
+    (
+        'td_brute_force',
+        'bu_brute_force',
+        'td_exhaustive_no_of_etas',
+        'bu_exhaustive_no_of_etas',
+        'td_exhaustive_block_structure',
```

Comment:
We need more eyes on the new naming. @stellabelin could you comment?

Reply:
I think the current naming is a bit confusing, but not sure what naming would make things clearer. Some thoughts:

- I think that it's a bit confusing that we both have `brute_force` and `exhaustive` in the naming, to me it's not apparent what the difference between them are from the name. 
- Bottom up is not a true exhaustive right? Since it stops when no better model is found? Perhaps `stepwise` would be useful here? 
- In general I find top down/bottom up a bit confusing here, given that `bu_brute_force` still uses the top bottom approach for the block structure (Idk, maybe in the future we'll have a bottom up approach for that too).
- The abbreviation `td` and `bu` is a bit confusing, however using the full name would render the names of these algorithms very long. 

Not sure what a good alternative to naming would be, I was otherwise thinking that we could add an option, so that `algorithm: Literal['top_down_exhaustive', 'bottom_up_exhaustive']` and `eta_selection: Literal['variance', 'covariance', 'both']`, however more options is also more complex so not sure if this is a good idea.

Reply:
New naming has now been added using two arguments.
One main `algorithm` argument takes the values of `top_down_exhaustive` and `bottom_up_stepwise`. If possible, the same algorithm will be used in both the search for the number of etas and for the covariance structure. However, another option will be used if the chosen algorithm only exists for one of the two searches.

Then there also exists a second optional argument called `correlation_algorithm` which can define if a separate algorithm is to be used for the covariance structure search. (However, this search currently only supports `top_down_exhaustive`)